activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -985,13 +985,34 @@ def foreign_key_options(from_table, to_table, options) # :nodoc:
       end
 
       def dump_schema_information #:nodoc:
+        versions = ActiveRecord::SchemaMigration.order('version').pluck(:version)
+        insert_versions(versions)
+      end
+
+      def insert_versions(versions)
```

Comment:
This method doesn't actually insert the versions. It returns the SQL that's executed to perform the inserts. Let's name the method to reflect that it's returning insert SQL.